README.md

```diff
@@ -1463,7 +1468,8 @@ __Example:__
     })
 
     const order = await client.checkout.shippingMethods({
-      bearerToken: token.access_token
+      bearerToken: token.access_token,
+      { include: 'shipping_rates' }
```

Comment:
Shouldn't it be `client.checkout.shippingMethods({bearerToken: token.access_token}, {include: 'shipping_rates'})`?